src/pybreaker.py

```diff
@@ -5,15 +5,30 @@
 For more information on this and other patterns and best practices, buy the
 book at https://pragprog.com/titles/mnee2/release-it-second-edition/
 """
+from __future__ import annotations
```

Comment:
This is a future flag, you can read more about it on:
- https://lukasz.langa.pl/61df599c-d9d8-4938-868b-36b67fdb4448/
- https://docs.python.org/3/library/__future__.html

But as a summary... It makes the interpreter to not see the type annotations at runtime. Meaning that we gain performance, and we can use syntax of further Python versions (as the IDE is prepared for it).